### PR TITLE
WLSSB patch 1.1

### PR DESCRIPTION
**Pull request by Lycanium-Z**
Nerfed and buffed a few people (mainly nerfed)

Added Bunnery5 and Revamped c7

### CODE REVIEW

**Comment by Lycanium-Z** [comment]
Oh yeah i have to update the data text as well >_>
Well it shouldnt be that bad

File: mods/wlssb/abilities.js [hunk 1]
```diff
@@ -327,4 +318,13 @@ exports.BattleAbilities = {
 			}
 		},
 	},
+	//bunnery5
+	muscles: {
+		id: "muscles",
+		name: "Muscles",
+		desc: "+2 defense, +2 Special defense, -3 attack, +1.5 special attack on switch in.",
+		onStart: function (pokemon) {
+			this.boost({atk: -4, def: 2, spa:1, spd: 2});
```
Comment (DeathlyPlays) [inline mods/wlssb/abilities.js]: Space after the spa:

File: mods/wlssb/moves.js [hunk 2]
```diff
@@ -423,22 +454,23 @@ exports.BattleMovedex = {
 	//Stabby the Krabby
 	"stabstab": {
 		category: "Physical",
-		basePower: 100,
-		accuracy: true,
-		desc: 'Always hits, hits twice, 25% chance to flinch.',
+		basePower: 60,
+		accuracy: 100,
+		desc: 'hits twice, 20% chance to flinch.',
```
Comment (DeathlyPlays) [inline mods/wlssb/moves.js]: Capitalize H

File: mods/wlssb/moves.js [hunk 3]
```diff
@@ -91,6 +92,7 @@ exports.BattleMovedex = {
 			this.attrLastMove('[still]');
 			this.add('-anim', source, "Surf", target);
 		},
+		flags: {protect: 1, mirror: 1},
```
Comment (DeathlyPlays) [inline mods/wlssb/moves.js]: P sure this was intentional

File: mods/wlssb/moves.js [hunk 4]
```diff
@@ -99,41 +101,61 @@ exports.BattleMovedex = {
 		type: "Water",
 	},
 	// C733937 123
-	lightshotgigalance: {
```
Comment (DeathlyPlays) [inline mods/wlssb/moves.js]: Missing a desc

File: mods/wlssb/moves.js [hunk 5]
```diff
@@ -736,4 +739,50 @@ exports.BattleMovedex = {
 		zMoveBoost: {spa: 1},
 		contestType: "Clever",
 	},
+	//bunnery5
+	bunneryhatesyouseed: {
+		category: "status",
```
Comment (DeathlyPlays) [inline mods/wlssb/moves.js]: Capitalize S

File: mods/wlssb/statuses.js [hunk 6]
```diff
@@ -144,8 +144,9 @@ exports.BattleStatuses = {
 	},
 	c733937123: {
 		exists: true,
-		onStart: function () {
+		onStart: function (pokemon) {
 			this.add('c', '@C733937 123', 'Hello opponent, Welcome to Spacial Bros, I, C733937 123, shall defeat you.....hopefully.');
+			this.useMove('psycheup', pokemon);
```
Comment (DeathlyPlays) [inline mods/wlssb/statuses.js]: Its Psych Up not Psyche Up

File: mods/wlssb/statuses.js [hunk 7]
```diff
@@ -165,6 +166,13 @@ exports.BattleStatuses = {
 		onFaint: function (pokemon) {
 			this.add('c', '+Auction', 'Ya know, I think I should\'ve gotten __burn everything__ as my ability ;_;');
 		},
+		onTryHit: function (target, source, move) {
+			if (target !== source && (move.type === 'Fire' || move.type === 'Water')) {
+				move.accuracy = true;
+				this.add('-immune', target, '[msg]', '[from] ability: Duality');
```
Comment (DeathlyPlays) [inline mods/wlssb/statuses.js]: His ability says its Stance Change

**Comment by DeathlyPlays** [comment]
Probaby should've started a review mb fam

File: mods/wlssb/statuses.js (hunk 7 above)
Comment (Lycanium-Z) [reply mods/wlssb/statuses.js]: Yes, but it itll be his innate ability

File: mods/wlssb/moves.js (hunk 4 above)
Comment (Lycanium-Z) [reply mods/wlssb/moves.js]: Done

**Comment by HoeenCoder** [comment]
Ping me when its all tested and ready

**Comment by RagiingAlways** [comment]
~~Never HH, It will ALWAYS BE ADDING MORE~~
:P

**Comment by Lycanium-Z** [comment]
Ok thats the last of the updates!

everything is tested and works!

Will work on Descriptions soon @hoeencoder

**Comment by HoeenCoder** [comment]
ok ping me when this is mergable

**Comment by Lycanium-Z** [comment]
Im gonna make a new PR cus c9 is being a bitch rn

**Pull request closed without merging**